README.md

```diff
@@ -1,2 +1,45 @@
-# hw01
-Repository for homework 01
+---
+title: "HW1"
+author: "Sanja Miklin"
+date: "10/1/2018"
+output: html_document
+---
+
+# My Bio
+
+---
+
+> Everything
+> is better upside down!
+
+![This is me!](images/railway.jpg)  
```

Comment:
The image is probably unshown because you haven't pushed it to the GitHub server or because it doesn't exist on a website and has an HTML link of its own. I think you could copy the file path of any of your photos on Facebook and attach it instead.

Reply:
Hey,
thanks for the feedback! But I'm surprised the picture is not showing—would you mind taking a screenshot of what you're seeing?
It renders fine for me
![image](https://user-images.githubusercontent.com/30011000/46688497-21260080-cbc3-11e8-8d75-207d9c4e4b18.png)